app/javascript/dashboard/mixins/specs/conversaton.spec.js

```diff
@@ -0,0 +1,103 @@
+import conversationMixin from '../conversations';
+import conversationFixture from './conversationFixtures';
+import commonHelpers from '../../helper/commons';
+commonHelpers();
+
+describe('#conversationMixin', () => {
+  it('should return unread message count 2 if conversation is passed', () => {
+    expect(
+      conversationMixin.methods.unreadMessagesCount(conversationFixture)
+    ).toEqual(2);
+  });
+  it('should return last message if conversation is passed', () => {
+    const lastMessage = {
+      id: 438100,
+      content: 'Hey',
+      account_id: 1,
+      inbox_id: 37,
+      conversation_id: 5815,
+      message_type: 0,
+      created_at: 1621145476,
+      updated_at: '2021-05-16T05:48:43.910Z',
+      private: false,
+      status: 'sent',
+      source_id: null,
+      content_type: 'text',
+      content_attributes: {},
+      sender_type: null,
+      sender_id: null,
+      external_source_ids: {},
+    };
```

Comment:
the data could be written in a separate file so that it is easy to read the tests.